app/src/main/res/layout/admin_auth_fragment.xml

```diff
@@ -1,7 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <layout xmlns:android="http://schemas.android.com/apk/res/android">
-
   <androidx.constraintlayout.widget.ConstraintLayout
+    android:id="@+id/admin_auth_container"
     android:layout_width="match_parent"
-    android:layout_height="match_parent"></androidx.constraintlayout.widget.ConstraintLayout>
+    android:layout_height="match_parent"
+    android:background="@color/white">
+    <TextView
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:text="Admin Auth" />
```

Comment:
Please move to strings.xml.

Reply:
This is temporary text that is removed in the next PR.